### DISCUSSION TITLE
Export to PDF or HTML

### DISCUSSION THREAD

**Discussion by blank-manash** [iamcco/markdown-preview.nvim#446] [Q&A]
Is there a way to directly export what we are seeing to HTML or PDF?

**Comment by Vaisakhkm2625** [comment]
Yes... :)

For PDF
---
it is simple as pressing CTRL+P in any browser 
it will show printing preview...  then press save as PDF

or you can press 3 dots or line on the top right corner and find print option

For HTML
---
Right click on page, select save page as.... save

or you can press 3 dots or line on the top right corner and find save page as option.

**Reply by Vaisakhkm2625** [reply]
if you want any other formats like LaTeX, you can use pandoc utililty
